spec/index.md

```diff
@@ -2376,6 +2376,17 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `bitcoin_get_block_headers` {#ic-bitcoin_get_block_headers}
```

Comment:
Could you please mark this method as "experimental": "The bitcoin get block headers management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way."

Reply:
Done!